sovtoken/sovtoken/test/helper.py

```diff
@@ -43,7 +44,7 @@ def send_xfer(looper, inputs, outputs, sdk_pool_handle, extra_data=None):
 
 def check_output_val_on_all_nodes(nodes, address, amount):
     for node in nodes:
-        handler = node.write_manager.request_handlers[XFER_PUBLIC][0]
+        handler = next(h for h in node.write_manager.request_handlers[XFER_PUBLIC] if isinstance(h, XferHandler))
```

Comment:
please, use get_handler_by_type_wm helper method from indy-plenum for this

Reply:
fixed